lms/envs/common.py

```diff
@@ -714,7 +714,16 @@
     # .. toggle_tickets: https://openedx.atlassian.net/browse/SOL-1325
     'ENABLE_OPENBADGES': False,
 
-    # Enable LTI Provider feature.
+    # .. toggle_name: FEATURES['ENABLE_LTI_PROVIDER']
+    # .. toggle_implementation: DjangoSetting
+    # .. toggle_default: False
+    # .. toggle_description: When set to True, Open edX site can be used as an LTI Provider to other systems
+    #    and applications.
+    # .. toggle_warnings: After enabling this feature flag you need to run database migrations which will create
+    #    database tables required for LTI provider functionality.
```

Comment:
There are a lot more steps involved with setting up edX as an LTI provider. It might be easiest to link to the docs at https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/lti/index.html

Reply:
Added docs link